index.js

```diff
@@ -92,6 +93,16 @@ module.exports = {
     // rely on implicit returns when return booleans or undefined.
     'consistent-return': ['error', { treatUndefinedAsUnspecified: true }],
     'eol-last': ['error', 'always'],
+
+    'import/order': [
+      'warn',
+      {
+        alphabetize: { order: 'asc', caseInsensitive: true },
+        'newlines-between': 'always',
+        pathGroups: [{ pattern: '{,~}/**', group: 'internal' }],
+        groups: ['builtin', 'external', ['internal', 'parent', 'sibling'], 'index'],
```

Comment:
also niceee

Reply:
For `internal` can we also group `@mixmaxhq` imports together?

Reply:
As a separate section above the `'internal', 'parent', 'sibling'` section?

Reply:
Seems like it might be nice to group `/utils` and `@mixmaxhq/*` together, maybe, and in a separate section?

Reply:
Updated!